lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -160,7 +160,7 @@
     }
 
     &.action-vote {
-
+       display: inline-block !important;
```

Comment:
@adampalay yeah might be there is a way. but the inline CSS apply through js on runtime. which create the issue so that's why I used this.